### PR TITLE
Support for limitrange k8s object

### PR DESCRIPTION
**Pull request by Rohit-PX**
This is required for deploying and validating ingress controllers (specifically nginx ingress controllers)

Signed-off-by: Rohit-PX <rohit@portworx.com>

### CODE REVIEW

File: k8s/core/limitrange.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
+package core
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (saheienko) [inline k8s/core/limitrange.go:5]: nit: `v1` and `corev1` name aliases point  to the same import.

File: k8s/core/limitrange.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+)
+
+// LimitRangeOps is an interface to perform k8s LimitRange operations
+type LimitRangeOps interface {
+	// GetLimitRange gets the secrets object given its name and namespace
```
Comment (saheienko) [inline k8s/core/limitrange.go:12]: nit: replace `secrets object` with `limitrange object` for comments.

File: k8s/core/limitrange.go (hunk 1 above)
Comment (Rohit-PX) [reply k8s/core/limitrange.go]: fixed.

File: k8s/core/limitrange.go (hunk 2 above)
Comment (Rohit-PX) [reply k8s/core/limitrange.go]: fixed.

**Pull request merged**